src/cart/crt_iv.c

```diff
@@ -410,20 +410,21 @@ crt_ivf_finalize(struct iv_fetch_cb_info *iv_info, crt_iv_key_t *iv_key,
 
 			output = crt_reply_get(rpc);
 			output->ifo_rc = output_rc;
-
+			output->ifo_misc = *output_value;
 			rc = crt_reply_send(rpc);
+			memset(&output->ifo_misc, 0, sizeof(output->ifo_misc));
```

Comment:
not sure why you do memsets of those in number of places. those shouldnt be needed

Reply:
oh, this is iv_value, if I do not reset it here, then RPC destroy will try to destory the value here.

Reply:
ugh yeah i see, this feels very hacky; probably more reasons to not override iv_value to avoid situations like these